app/customer.py

```diff
@@ -0,0 +1,47 @@
+from math import sqrt
+
+from app.car import Car
+from app.shop import Shop
+
+
+class Customer:
+    def __init__(self,
+                 name: str,
+                 product_cart: dict,
+                 location: list[int],
+                 money: float,
+                 car: dict) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = Car(car["brand"], car["fuel_consumption"])
```

Comment:
The 'Car' class is being instantiated with 'car["brand"]' and 'car["fuel_consumption"]'. Ensure that the 'car' dictionary always contains these keys to avoid potential KeyError.